### PR TITLE
Volcano Part Cooling Shroud for AB

### PR DESCRIPTION
**Pull request by perikanttila**
* [x ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Original E3D V6 part cooling shroud was the source
#### How was it tested? 
Printed it our
#### Any background context you want to provide?
no
#### Screenshots (if appropriate)
no
#### Further notes
no

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -138,5 +138,6 @@ like so:
 | | [V0 Electronics Cover](./xbst_/V0_Electronic_Panel)| A printed cover for v0 electronics. Requires larger printer to print. | :heavy_check_mark: | :x: | :x: | :x: |
 | | [V0 PiCam Mount](./xbst_/V0_Picam)| A Raspberry Pi Camera Mount For V0 | :heavy_check_mark: | :x: | :x: | :x: |
 | | [DuetZero](./xbst_/DuetZero) | Pi Zero mount over a Duet2 for space saving | :heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: |:x:|
+|karip|[Volcano Shroud](./karip/volcano_shroud)|Vocano Shroud|:x:|:x:|:heavy_check_mark:|:grey_question:|
```
Comment (KiloQubit) [inline printer_mods/README.md:141]: Please move this line up so that it's in alphanumeric order by username.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your PR.  There are just two small changes needed.

Please remove the space from the STL filename, e.g. rename it to `Volcano_Shroud.stl`

**Comment by KiloQubit** [comment]
Do you intend to update this PR to address the review comments?

**Comment by perikanttila** [comment]
I will, it has been a bit busy lately

**Comment by KiloQubit** [comment]
Hi, are you still planning to update this mod?

**Comment by perikanttila** [comment]
Sorry yes, i was moving and not able to update will do it soon
Thanks

**Comment by bryanboettcher** [comment]
@perikanttila I'm willing to take over the remaining housekeeping on this PR if you'd be OK with that.

**Comment by perikanttila** [comment]
Thank you.

**Comment by bdbell** [comment]
This still requires the one review item be addressed.  Will it be addressed?

**Comment by Rhastlyn** [comment]
This is one is quite old, anyone intend to complete this PR?

**Comment by perikanttila** [comment]
i hope someone does complete it. i can not do it. no skills to complete it

...
Kari

**Comment by cbc02009** [comment]
waiting on SB release to revisit finalizing this mod.

**Comment by cbc02009** [comment]
Closing due to inactivity.

**Pull request closed without merging**